python/cudf/cudf/core/series.py

```diff
@@ -539,7 +545,21 @@ def __init__(
                 data = {}
 
         if not isinstance(data, ColumnBase):
-            data = column.as_column(data, nan_as_null=nan_as_null, dtype=dtype)
+            has_cai = (
+                type(
```

Comment:
Please add a comment explaining why we need to use `gettatr_static`.

Reply:
Added a comment, explaining the same.